data/phi0474/phi010/phi0474.phi010.perseus-eng2.xml

```diff
@@ -46,12 +46,12 @@
       <editor role="translator">C. D. Yonge</editor>
       <imprint>
        <pubPlace>London</pubPlace>
-       <publisher>George Bell &amp; Sons</publisher>
+       <publisher>Henry G. Bohn, York Street, Covent Garden</publisher>
```

Comment:
This is ok, but I don't think we do it elsewhere for Bohn. Should it be expanded elsewhere?

Reply:
hi @lcerrato .This is the string for the publisher you had listed in the comments for the last Cicero pull request, I was wondering about that. Would you say I should just go with Henry G. Bohn, and that's what I will use for this file and the other English Cicero files.

Reply:
@AlisonBabeu 
Hi, yes, I have no idea why I would suggest a full address, but it would be a good idea to check globally and pick whatever format you think is appropriate.

Reply:
thanks @lcerrato I'm closing this one due to a GitHub issue on my end and have opened what I hope is the final pull request on this issue here #541.